### PR TITLE
Update doc with the new API DOM.select(selector).events(eventType)

### PR DESCRIPTION
**Pull request by ctranxuan**
Hello,
Please find the update of the documentation in accordance of the new API (see https://github.com/cyclejs/cycle-dom/releases/tag/v5.1.0).

Hope this helps,
Cédric.

### CODE REVIEW

File: _posts/2015-01-31-getting-started.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ import {makeDOMDriver, h} from '@cycle/dom';
 
 function main(drivers) {
   return {
-    DOM: drivers.DOM.get('input', 'click')
+    DOM: drivers.DOM.select('input').events('click')
```
Comment (staltz) [inline _posts/2015-01-31-getting-started.md:84]: There is still a mention of `DOM.get()` a paragraph below this

**Comment by ctranxuan** [comment]
Sorry, fixed the missing onee

**Comment by staltz** [comment]
Thanks for the PR

**Comment by ctranxuan** [comment]
You're welcome!

**Pull request merged**